filament/src/fg/ResourceAllocator.cpp

```diff
@@ -40,14 +76,92 @@ void ResourceAllocator::destroyRenderTarget(RenderTargetHandle h) noexcept {
     return mBackend.destroyRenderTarget(h);
 }
 
-TextureHandle ResourceAllocator::createTexture(SamplerType target, uint8_t levels,
-        TextureFormat format, uint8_t samples, uint32_t width, uint32_t height,
-        uint32_t depth, TextureUsage usage) noexcept {
-    return mBackend.createTexture(target, levels, format, samples, width, height, depth, usage);
+backend::TextureHandle ResourceAllocator::createTexture(const char* name,
```

Comment:
Consider renaming to `acquireTexture` since it does not necessarily create a brand new texture.